README.md

```diff
@@ -210,34 +210,33 @@ const MyComponent = () => {
 - Excellent centralized server-side data hydration support
 - Shared Cache between Micro Frontend Holocron Modules
 - Immutable Redux State
+- Minimal configuration
 
 **Cons**
 - More steps and dependencies
 
 ```
-npm i -S fetchye redux-immutable fetchye-redux-provider fetchye-immutable-cache
+npm i -S fetchye fetchye-one-app
```

Comment:
Should we mention `redux` and `react-redux`

Reply:
redux and react-redux are provided as part of one-app so i think its okay not to